packages/UI/src/components/organisms/cards/StyledCommonComponents.tsx

```diff
@@ -2,7 +2,7 @@ import {
   motion,
   LazyMotion,
   domAnimation,
-} from 'framer-motion/dist/framer-motion';
+} from 'framer-motion'; // import from /dist/framer-motion is not working, import from framer-motion is working
```

Comment:
some members have this working with `/dist` so can you please check if this doesn't break the UI library by running `yarn workspace @devlaunchers/components build-storybook` and see it succeeds

Reply:
I remember experiencing issues while using yarn install was storybook part of the dependencies. If yes can I do yarn add @storybook/react?